scripts/release.sh

```diff
@@ -342,13 +342,15 @@ main() {
     local release_url
     local gh_release_args=()
 
-    # For the main branch (v3.6), we should mark the release as a prerelease.
-    # The release-3.5 (v3.5) branch, should be marked as latest. And release-3.4 (v3.4)
-    # should be left without any additional mark (therefore, it doesn't need a special argument).
-    if [ "${BRANCH}" = "main" ]; then
-      gh_release_args=(--prerelease)
-    elif [ "${BRANCH}" = "release-3.5" ]; then
-      gh_release_args=(--latest)
+    local prompt
+    read -p "Mark this as a the latest release [y/N]? " -r prompt
+    if [[ "${prompt,,}" == "y" ]]; then
+      gh_release_args=(--latest=true)
+    else
+      read -p "Mark this as a pre-release [y/N]? " -r prompt
```

Comment:
Same as above, would be good to inform that is expectation about what is and what shouldn't be a pre-release.

Reply:
Done